cellmlmanip/model.py

```diff
@@ -232,10 +245,11 @@ def get_rdf_value(self, subject, predicate):
 
         Note: expects exactly one triple to match and the result to be a literal. Its string value is returned.
         """
-        triples = list(self.get_rdf_annotations(subject, predicate))
-        assert len(triples) == 1
-        assert isinstance(triples[0][2], rdflib.Literal)
-        value = str(triples[0][2]).strip()  # Could make this cleverer by considering data type if desired
+        triples = self.get_rdf_annotations(subject, predicate)
+        triple = next(triples)
```

Comment:
Not sure why these changes are in this PR. But this will raise an iterator error if triples is empty, no? So I think `assert len(triples) == 1, 'Expecting exactly 1 triple'` would be better?

Reply:
Good point I changed it to.
```
        triple = next(triples, None)
        assert triple is not None, 'Expecting exactly 1 triple'
```
To catch that case it's just that I want to avoid unnecessary list creation.

Reply:
```
>>> next(iter([]))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
StopIteration
```

Reply:
Ah ok. Just spotted the `None`

Reply:
Hmmmm, what about `triples = [None]`? That triggers the same assert?

Reply:
I just checked rdflib